e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -123,6 +123,7 @@ if (-not ($keyVaultName -match "^[a-zA-Z][a-zA-Z0-9-]{1,22}[a-zA-Z0-9]$"))
 # New certs will be generated each time you run the script as the script cleans up in the end
 ########################################################################################################
 
+$subjectPrefix = "IoT Test"
```

Comment:
This was previously not initialized causing the wrong certs to get deleted during cleanup